guides/source/testing.md

```diff
@@ -252,7 +252,7 @@ NameError: undefined local variable or method 'some_undefined_variable' for #<Ar
     test/models/article_test.rb:11:in 'block in <class:ArticleTest>'
 
 
-rails test test/models/article_test.rb:9
+bin/rails test test/models/article_test.rb:9
```

Comment:
Ditto

Reply:
@eugeneius I updated my PR to print `bin/rails test ...`